etc/infrasim.yml.default

```diff
@@ -47,16 +47,15 @@ compute:
     storage_backend:
         #Set drive list and define drive attributes
         -
-            controller:
-                type: ahci
-                max_drive_per_controller: 8
```

Comment:
How about keep it? and add a comment that "for ahci, maximium allowable number is 6".